tests/integration/test_metadata.py

```diff
@@ -0,0 +1,32 @@
+import json
+import pytest
+
+
+def ensemble(client):
+    # generate an ensemble
+    resp = client.post_check("/experiments", json={"name": "test_metadata"})
+    exp_id = resp.json()["id"]
+    resp = client.post_check(
+        f"/experiments/{exp_id}/ensembles", json={"parameters": []}
+    )
+    ens_id = resp.json()["id"]
+    return f"/ensembles/{ens_id}"
+
+
+def post_record(client, name="rec_test", data="[1, 2, 3, 4, 5]"):
+    # generate a record
+    ens_url = ensemble(client)
+    rec_url = f"{ens_url}/records/{name}"
+    client.post_check(
+        f"{rec_url}/matrix",
+        data=data,
+    )
+    return rec_url
+
+
+def test_record_metadata(client):
+    rec_url = post_record(client)
+    metadata_in = dict(clusters=dict(user1=1))
```

Comment:
`metadata_in = {"clusters": {"user1": 1}}`?
I tend to prefer dictionary literals for a few reasons, but we can debate this.

Reply:
This should be unified across the repo, either constructor or literals all over the place :)

Reply:
I don't really have a preference but since it's used in `test_observations.py` I thought we should be consistent.